frontend/src/metabase/dashboard/hooks/use-dashboard-url-params.ts

```diff
@@ -87,9 +92,27 @@ export const useDashboardUrlParams = ({
     titled,
     font,
     setFont,
-    theme,
+    theme: normalizedTheme,
     setTheme,
     hideParameters: hide_parameters,
     hideDownloadButton: hide_download_button,
   };
 };
+
+/**
+ * When both `background: false` and `theme: "transparent"` options are supplied,
+ * the new behavior takes precedence (metabase#43838)
```

Comment:
[The spec is here.](https://www.notion.so/metabase/Re-design-transparent-theme-for-static-embedding-and-public-links-a28e98a6e5c64301a48387c7b333d569?pvs=4#2e1eff4f88df49e3b6ee7e37af559ea1)